00_introduction/21_architecture_root_server.txt

```diff
@@ -2,58 +2,45 @@
 
 === Specifications for Rudder Root Server
 
-==== Hardware specifications
+==== Hardware specifications and sizing
 
-A dedicated server is strongly recommended to deploy Rudder Root Server.
+A dedicated server is strongly recommended, either physical or virtual with at least one dedicated core.
+Rudder Server is running on both 32 (if available) and 64 bit versions of every supported Operating System.
 
-Your Rudder Root Server can be either a physical or a virtual machine. 
+===== Memory
 
-Rudder Server is running on both 32 (if available) and 64 bit versions of every supported
-Operating System. 
+The required amount of RAM mainly depends on the number of managed nodes. A general rule for the minimal value is:
 
-Any CPU provided on commodity servers available nowadays should be fine. On
-a VM server, you should dedicate at least one core to the Rudder server.
+* less than 50 nodes: 2GB
+* between 50 and 1000 nodes: 4GB
+* more than 1000 nodes: 4GB + 1GB of RAM by 500 nodes above 1000.
 
-The amount of RAM needed by the server depends mainly on the number of managed nodes.
-
-The base requirement is to have a server with 2GB of RAM. 
-On production environment, when managing more than 50 nodes, it is recommended to 
-dedicate at least 4GB of RAM to the server.
+When managing more than 1000 nodes, we also advice you to use a multiserver
```

Comment:
I see you changed "advise" to "advice" here. This is actually incorrect (advice is the noun, and advise is the verb, so "advise" is right here). However, it may be better to replace it with "recommend".